classes/music_album.rb

```diff
@@ -7,4 +8,8 @@ def initialize(publish_date, on_spotify, archived)
     super(publish_date, archived)
     @on_spotify = on_spotify
   end
+
+  def can_be_archived?
+    (Date.today.year - @publish_date.year) > 10 && @on_spotify == true
```

Comment:
You can call the method in the parent class using the **super** keyword. Hence, you can rewrite this as 
```
  def can_be_archived?
    return true if super && @on_spotify

    false
  end
```
  
  But your logic is still accurate.

Reply:
okay thanks